.buildkite/pipeline.gpu_large.yml

```diff
@@ -49,3 +49,16 @@
     - pip install -Ur ./python/requirements/ml/requirements_ml_docker.txt
     - ./ci/env/env_info.sh
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=gpu,-timeseries_libs,-py37,-post_wheel_build doc/...
+
+- label: ":tv: :python: Lightning 2.0 Train GPU tests"
+  conditions:
+    ["NO_WHEELS_REQUIRED", "RAY_CI_TRAIN_AFFECTED"]
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
+    - NO_DASHBOARD=1 ./ci/env/install-minimal.sh 3.8
```

Comment:
Why is the `NO_DASHBOARD` introduced?

Reply:
The background is that the GPU images are not made to rebuild Ray - they don't necessarily contain all required dependencies. Kai fixed this pipeline by just not to build the dashboard.